projects/fizzbuzz/java/FizzBuzz.java

```diff
@@ -0,0 +1,37 @@
+package fizzbuzz;
+
+import java.util.Scanner;
+
+public class FizzBuzz
+{
+    public static void main(String[] args)
+    {
+        Scanner input = new Scanner(System.in);
+        
+        //User enters a number
+        System.out.println("Enter a number:");
+        int amount = input.nextInt();
+        
+        //Print the numbers
+        for (int i = 1; i <= amount; i++)
+        {
+            //If number is divisible by 3
+            if (i%3 == 0)
+            {
+                System.out.print ("Fizz");
+            }
+            //If number is divisible by 5
+            if (i%5 == 0)
+            {
+                System.out.print ("Buzz");
+            }
+            //If not divisible by 3 or 5
+            if (i%3 != 0 && i%5 != 0)
+            {
+                System.out.print (i);
+            }
```

Comment:
Solution should work as is. However, there is one way to improve efficiency: Currently, all three 'if' statements will be checked every time. You could try something like:

```
if (i%3 != 0 && i%5 != 0){
    //do stuff
}
else {
    if (i%3 == 0){
        //do stuff
    }
    if (i%5 == 0){
        //do stuff
    }
}
```
and then you'd avoid running code that isn't necessary. Pretty small difference in this particular example, but imagine if we ran this for some n = 1,000,000,000,000+ or something like that.  -Nick Hager

Reply:
Thanks for the suggestion! I updated it with your ordering.